src/wallet/rpc/wallet.cpp

```diff
@@ -809,6 +809,61 @@ static RPCHelpMan migratewallet()
     };
 }
 
+RPCHelpMan getxpub()
+{
+    return RPCHelpMan{"getxpub",
+                "Returns the xpub most recently used to generate descriptors for this descriptor wallet. "
+                "Not entirely useful right now as it returns the xpub of the root, and there are "
```

Comment:
in 9900207360a2852041ceed260c3430e92ff860c5

The comment is a bit outdated it seems, since the method can also return xprv.

Reply:
Dropped